src/kern/npf_impl.h

```diff
@@ -511,6 +534,8 @@ void		npf_state_setsampler(void (*)(npf_state_t *, bool));
 void		npf_setkernctx(npf_t *);
 npf_t *		npf_getkernctx(void);
 
+size_t		npf_pptp_gre_hdr_len(void);
```

Comment:
- There is `npf_pptp_conn_conkey()` declared above.  Generally, it is a good idea to group the functions together, in this case  `npf_pptp_*()`.
- But also see a question below.